.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Enhancement Request
```

Comment:
this is not matching with feature-request.md of other repos PRs

Reply:
Updated